surfsense_backend/app/services/document_processing/docling_service.py

```
@@ -0,0 +1,346 @@
+        if len(content) <= LARGE_DOCUMENT_THRESHOLD:
+            # For smaller documents, use direct processing
+            logger.info(f"📄 Document size: {len(content)} chars - using direct processing")
+            from app.prompts import SUMMARY_PROMPT_TEMPLATE
+            summary_chain = SUMMARY_PROMPT_TEMPLATE | llm
+            result = await summary_chain.ainvoke({"document": content})
+            return result.content
+        
+        logger.info(f"📚 Large document detected: {len(content)} chars - using chunked processing")
+        
+        # Import chunker from config
+        from app.config import config
```

Ruff (0.12.2)

`app.config.config` imported but unused

Remove unused import: `app.config.config`

(F401)